src/common/components/featureFlags/FeatureFlagsContext.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { createContext, useContext, ReactNode } from 'react';
+
+type FeatureFlagsProviderProps = {
+  children: ReactNode;
+};
+
+export type FeatureFlagsContextProps = {
+  publicHankkeet: boolean;
+  hanke: boolean;
+  accessRights: boolean;
+};
+
+const FeatureFlagsContext = createContext<FeatureFlagsContextProps | undefined>(undefined);
+
+export function FeatureFlagsProvider({ children }: FeatureFlagsProviderProps) {
+  const value: FeatureFlagsContextProps = {
```

Comment:
Sonar seems to have an issue with this:

The 'value' object passed as the value prop to the Context provider changes every render. To fix this consider wrapping it in a useMemo hook.)

So maybe something like:

```
  const value: FeatureFlagsContextProps = useMemo(
    () => ({
      publicHankkeet: window._env_.REACT_APP_FEATURE_PUBLIC_H === '1',
      hanke: window._env_.REACT_APP_FEATURE_H === '1',
      accessRights: window._env_.REACT_APP_FEATURE_ACCESS_RIGHTS === '1',
    }),
    []
  );
```

Reply:
I'm not sure if the dependency array should be empty or not.

Reply:
Empty in this case as there are no dependencies.

Reply:
Wrapped the value in useMemo.